client.go

```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/omise/omise-go/internal"
 )
 
+// LibraryVersion to include in user agent
+var LibraryVersion = "1.0.12"
```

Comment:
@ajzkk I think you can just make this `libraryVersion` so it won't be exported.

Reply:
Updated